### ISSUE TITLE
Client Crash: Java: Java2D Queue Flusher

### ISSUE THREAD

**Issue by GodsRevelation** [runelite/runelite#11246]
**Describe the bug**
When using RuneLite, the client will unexpectedly crash. There's not much reason as to why. This occurs at any point while using the client (i.e. Opening for the first time/been using for hours). 

**To Reproduce**
No way to really reproduce that I know of.

**Expected behavior**
Client Crashes.

**Screenshots**
![Screen Shot 2020-04-10 at 5 52 00 PM](https://user-images.githubusercontent.com/63478329/79026205-0f297300-7b56-11ea-998f-b1f983ee7526.png)

I have copied the entire code that comes up, but was unsure if it was required. I couldn't find out too much about what the crash error was.

**Environment (please complete the following information):**
 - OS: Mac OS X 10.13.1
 - RuneLite version: 2.0.0

**Additional context**
This is what I could find in the logs about the most recent crash (which add the same error messages for the other recent ones as well).
![Screen Shot 2020-04-10 at 6 28 51 PM](https://user-images.githubusercontent.com/63478329/79027177-45b4bd00-7b59-11ea-86e1-3caec745e22c.png)

Let me know if anymore information is needed to best fix this issue.

Thank you.

**Comment by Nightfirecat** [comment]
Can you please upload your `jvm_crash_pid` file(s) which are in your logs folder?

**Comment by GodsRevelation** [comment]
[jvm_crash_pid_5200.log](https://github.com/runelite/runelite/files/4463607/jvm_crash_pid_5200.log)

Here is the most recent one!

**Comment by abextm** [comment]
Try [disabling hardware acceleration](https://github.com/runelite/runelite/wiki/Disable-Hardware-Acceleration#macos)

**Comment by GodsRevelation** [comment]
@abextm I have followed the link you have provided, and opened RL using the macOS option. Do you know if this will open RL moving forward with hardware acceleration disabled? Or will I need to open this way in the future?

**Comment by abextm** [comment]
You would need to do that in the future too. You might be able to make a shell script that does that and run that instead, I don't use osx

**Comment by GodsRevelation** [comment]
Ok, that makes sense! I will continue to monitor to see if the problem persists.

**Comment by GodsRevelation** [comment]
It has been working quite well, a lot less frequent client crashes. I had only one yesterday and it seems to be the accelerator this time, according to the log. But other than that, I'm very pleased with how the disabling the hardware acceleration has helped thus far.

Here is the log of the crash yesterday (4/11/2020). Not sure if it of any use.
[jvm_crash_pid_5780.log](https://github.com/runelite/runelite/files/4467548/jvm_crash_pid_5780.log)

**Comment by GodsRevelation** [comment]
@Nightfirecat @abextm Do y'all have any other recommendations based on the previous crash logs? The disabling of the hardware acceleration seemed like it had fixed the problem, but the crashes have been happening quite frequent again.

I can supply the most recent logs, but they basically reflect my previous ones (as far as crashed thread/errors).

**Comment by abextm** [comment]
I'm not even sure why this code would be running if you have hwaccel disabled. Its deep in your graphics drivers, so maybe try updating those

**Comment by GodsRevelation** [comment]
Yeah I'm not sure. I did, however, update the macOS and made sure everything was up to date. The crashes seem to be more frequent and look similar (this is the crash from disabling the hardware accel)

[jvm_crash_pid_593.log](https://github.com/runelite/runelite/files/4498345/jvm_crash_pid_593.log)

**Comment by GodsRevelation** [comment]
I'm at a loss, can't seem to find a fix to the problem. I even started using the Vanilla client and it's having the same problems that RL is having (the same crash reason/log).
[jvm_crash_pid_619.log](https://github.com/runelite/runelite/files/4529044/jvm_crash_pid_619.log)

**Comment by GodsRevelation** [comment]
@Nightfirecat @abextm
I've been trying several different things to see if I could come up with a solution:
-I upgraded from 10.13.1 to 10.15.4 MacOS
-Re-enabled core dumps
-Replaced my old RAM, and doubled the capacity (4GB to 8GB)

And upon taking a look at all the crash logs, I'm getting the same Exception type/codes from other programs as well. The Vanilla client and RL both crash frequently, but for some reason OSbuddy isn't having this problem, I'm not sure why this is the case recently.
